src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -61,7 +62,6 @@ class ShowInterfacesSchema(MetaParser):
                 Optional('line_protocol'): str,
                 Optional('enabled'): bool,
                 Optional('connected'): bool,
-                Optional('err_disabled'): bool,
```

Comment:
May i ask why this line is removed?
this line is in place to catch outputs that has line protocol status i.e (err-disabled)

Reply:
It's a mistake, I didn't actively remove it

Reply:
Confirmed line is still there. No issue